spec/kakurenbo_puti/active_record_base_spec.rb

```diff
@@ -327,4 +327,33 @@
       end
     end
   end
+
+  describe '.soft_destroy_all' do
+    subject do
+      model_class.soft_destroy_all
+    end
+    let!(:model_instance) { model_class.create! }
+
+    it 'Soft-Delete model' do
+      expect {
+        subject
+      }.to change {
+        model_class.without_soft_destroyed.count
+      }.from(16).to(0)
+    end
```

Comment:
Order of Rspec test is not guaranteed.
So, this code should rewrite it as follows.

```ruby
expect {
  subject
}.to change {
  model_class.without_soft_destroyed.count
}.to(0)
```